### PR TITLE
Re-format the codebase (just style)

### PR DESCRIPTION
**Pull request by hearot**
Just re-format the code & add some licenses.

_wip, wip, I'm a sheep, say wip, wip, I'm a sheep!_

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -1,3 +1,27 @@
+# This file is a part of nylo
+#
+# Copyright (c) 2018 The nylo Authors (see AUTHORS)
```
Comment (hearot) [inline src/main.py:3]: Just found out that there's no `AUTHORS` file, I'm going to fix.

**Comment by hearot** [comment]
Just added AUTHORS & CONTRIBUTORS files. Do you agree with that, @veggero?
_wip, wip, I'm a sheep, say wip, wip, I'm a sheep!_

**Comment by veggero** [comment]
@hearot Well I don't agree I'll just delete that in next commit

**Comment by veggero** [comment]
@veggero IF* I don't agree, typo

File: src/main.py (hunk 1 above)
Comment (veggero) [reply src/main.py]: Please, I'd prefer not to have AUTHORS and CONTRIBUTORS files.

File: src/nylo/builtins.py [hunk 2]
```diff
@@ -1,36 +1,60 @@
+# This file is a part of nylo
```
Comment (veggero) [inline src/nylo/builtins.py:1]: builtins.py should not have the copyright. Only file beggining with __ shall have them.

File: src/main.py (hunk 1 above)
Comment (veggero) [reply src/main.py:3]: Please, I'd prefer not to have AUTHORS and CONTRIBUTORS files.

File: src/nylo/interpreter.py [hunk 3]
```diff
@@ -1,8 +1,31 @@
+# This file is a part of nylo
```
Comment (veggero) [inline src/nylo/interpreter.py:1]: interpreter.py should not have the copyright. Only file beggining with __ shall have them.

File: src/nylo/parser.py [hunk 4]
```diff
@@ -1,3 +1,27 @@
+# This file is a part of nylo
```
Comment (veggero) [inline src/nylo/parser.py:1]: parser.py should not have the copyright. Only file beggining with __ shall have them.

File: src/nylo/token.py [hunk 5]
```diff
@@ -1,18 +1,44 @@
+# This file is a part of nylo
```
Comment (veggero) [inline src/nylo/token.py:1]: token.py should not have the copyright. Only file beggining with __ shall have them.

File: src/nylo/tokens/keyword.py [hunk 6]
```diff
@@ -1,61 +1,86 @@
+# This file is a part of nylo
```
Comment (veggero) [inline src/nylo/tokens/keyword.py:1]: keyword.py should not have the copyright. Only file beggining with __ shall have them.

File: src/nylo/tokens/numstr.py [hunk 7]
```diff
@@ -1,72 +1,97 @@
+# This file is a part of nylo
```
Comment (veggero) [inline src/nylo/tokens/numstr.py:1]: numstr.py should not have the copyright. Only file beggining with __ shall have them.

File: src/nylo/tokens/struct.py [hunk 8]
```diff
@@ -1,24 +1,48 @@
+# This file is a part of nylo
```
Comment (veggero) [inline src/nylo/tokens/struct.py:1]: struct.py should not have the copyright. Only file beggining with __ shall have them.

File: src/nylo/tokens/symbol.py [hunk 9]
```diff
@@ -1,37 +1,60 @@
+# This file is a part of nylo
```
Comment (veggero) [inline src/nylo/tokens/symbol.py:1]: symbol.py should not have the copyright. Only file beggining with __ shall have them.

File: src/nylo/tokens/value.py [hunk 10]
```diff
@@ -1,65 +1,90 @@
+# This file is a part of nylo
```
Comment (veggero) [inline src/nylo/tokens/value.py:1]: value.py should not have the copyright. Only file beggining with __ shall have them.

**Review by veggero — CHANGES_REQUESTED** [review]
AUTHORS, CONTRIBUTORS files and some copyrights shall be deleted.

**Comment by veggero** [comment]
Also - could you switch from GPL to MIT?

**Comment by meloncello** [comment]
:(

**Comment by Knocks83** [comment]
@YellowPecora Best reason ever

**Pull request closed without merging**